LayoutTests/http/tests/notifications/notification.html

```diff
@@ -41,8 +43,17 @@
     assert_equals(notification.body, "test body");
     assert_equals(notification.tag, "test tag");
     assert_equals(notification.icon, "http://127.0.0.1:8000/notifications/foo.png");
+
+    let data1 = notification.data;
+    let data2 = notification.data;
+    assert_true(data1 === data2, "data getter should return same object");
```

Comment:
You really want a test where you gc() setTimeout() gc() again and then maybe sure the data stays the same.
Checking that the data stays the same without GC doesn't mean it will still be the case after GC.